chips/lowrisc/src/hmac.rs

```diff
@@ -224,35 +217,21 @@ impl<'a> hil::digest::DigestData<'a, 32> for Hmac<'a> {
     fn add_data(
         &self,
         data: LeasableBuffer<'static, u8>,
-    ) -> Result<usize, (ErrorCode, &'static mut [u8])> {
-        let regs = self.registers;
-
-        regs.cmd.modify(CMD::START::SET);
-
-        // Clear the FIFO empty interrupt
-        regs.intr_state.modify(INTR_STATE::FIFO_EMPTY::SET);
-
-        // Enable interrupts
-        regs.intr_enable.modify(INTR_ENABLE::FIFO_EMPTY::SET);
```

Comment:
These register accesses were deleted and I do not see anywhere else they have been inserted, is that intentional?